heft-plugins/heft-lint-plugin/src/LintPlugin.ts

```diff
@@ -114,9 +116,13 @@ export default class LintPlugin implements IHeftTaskPlugin<ILintPluginOptions> {
 
         // Hook into the changed files hook to collect the changed files and their programs
         accessor.onChangedFilesHook.tap(PLUGIN_NAME, (changedFilesHookOptions: IChangedFilesHookOptions) => {
+          // When using the TypeScript plugin, we need to determine the tsconfig path
+          // The default tsconfig path is used when not explicitly specified
+          const tsconfigPath: string = path.resolve(heftConfiguration.buildFolderPath, 'tsconfig.json');
```

Comment:
Although it is not documented on the public API typings, `changedFilesHookOptions.program.getCompilerOptions().configFilePath` is the resolved path to the tsconfig file used by the program, and simply needs to be converted to a relative path and be run through `Path.convertToSlashes`.

heft-typescript-plugin already uses this code path, so the custom extended type should already be defined in the codebase.